lib/zendesk_apps_support/installed.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'erubis'
+require 'json'
```

Comment:
Good question. It never worked! This files wasn't (completely) tested. But because I wrote a test for it, it started breaking `.to_json is not a function`.

Reply:
I don't think that is possible, as `installed.js` is being served correctly in production even without this change.

Reply:
It is very much possible. ZAT currently includes `require 'json'`, but when you run this in a test as standalone, it breaks.

Reply:
But this test exists for ZAS and it passes! https://github.com/zendesk/zendesk_apps_support/blob/master/spec/installed_spec.rb

Reply:
Because that tests doesn't run the `to_json` code.